cds_migrator_kit/rdm/migration/transform/xml_processing/rules/base.py

```diff
@@ -71,12 +71,85 @@ def title(self, key, value):
     return value.get("a", "TODO")
 
 
+abbreviations = []
+
+
+@model.over("description", "^246__")
+def description(self, key, value):
+    """Translates description."""
+    is_abbreviation = value.get("i") == "Abbreviation"
+    abbreviations.append(value.get("a"))
+
+    if is_abbreviation:
+        return "Abbreviations: " + "; ".join(abbreviations)
+
+    return
+
+
+@model.over("additional_descriptions", "(^500__)|(^520__)")
+@for_each_value
+@require(["a"])
+def additional_descriptions(self, key, value):
+    """Translates additional description."""
+    description_text = value.get("a")
+
+    if key == "500__":
+        additional_description = {
+            "description": description_text,
+            "type": {
+                "id": "other",  # what's with the lang
+            }
+        }
+    elif key == "520__":
+        additional_description = {
+            "description": description_text,
+            "type": {
+                "id": "abstract",  # what's with the lang
```

Comment:
> what's with the lang
I am not sure what does this comment mean?

Reply:
Sorry, this comment was more like "notes for myself" than a real question, that's why it looks confusing 😅  but what I meant was that if we create a record in CDS with additional description, it adds these translations of type and I'm not sure whether we want to keep this in the migrated records as well. 
![Screenshot 2024-09-16 at 14 28 51](https://github.com/user-attachments/assets/c0a6e95d-918b-47ca-85bb-9fd759e3f4ba)